README.md

```diff
@@ -182,7 +182,7 @@ SIG-specific GitHub discussions.
 | End-User SIG | Every other week on Thursday at 10:00 PT | [Google Doc](https://docs.google.com/document/d/1e-UNZA3Tuno9b53RQbe--whUcO0VIXF3P81oXsrBK6g) | [#otel-user-research](https://cloud-native.slack.com/archives/C01RT3MSWGZ) | [calendar-user-research](https://groups.google.com/a/opentelemetry.io/g/calendar-user-research) | [Daniel Gomez Blanco](https://github.com/danielgblanco) |
 | Security | Weekly on Wednesday at 08:30 AM PT | [Google Doc](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg) | [#otel-sig-security](https://cloud-native.slack.com/archives/C05A85QC281) | [calendar-sig-security](https://groups.google.com/a/opentelemetry.io/g/calendar-sig-security) | [Juraci Paixão Kröhling](https://github.com/jpkrohling) |
 | Project Tooling and Infrastructure | Weekly on Wednesday at 08:00 AM PT | [Google Doc](https://docs.google.com/document/d/1_dTP2XIaZoFjNiipkeMaGJN10NrhS20FiBTTJoQJeVM) | [#otel-project-tooling](https://cloud-native.slack.com/archives/C07BPU981PV) | [calendar-project-tooling](https://groups.google.com/a/opentelemetry.io/g/calendar-project-tooling) | [Austin Parker](https://github.com/austinlparker) |
-| Contributor Experience | TBA | [Google Doc](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0) | [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK) | [calendar-contributor-experience](https://groups.google.com/a/opentelemetry.io/g/calendar-contributor-experience) | [Juraci Paixão Kröhling](https://github.com/jpkrohling) |
+| Contributor Experience | Every Monday alternating between 9:00 and 17:00 UTC | [Google Doc](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0) | [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK) | [calendar-contributor-experience](https://groups.google.com/a/opentelemetry.io/g/calendar-contributor-experience) | [Juraci Paixão Kröhling](https://github.com/jpkrohling) |
```

Comment:
17:00 UTC will conflict with maintainer meeting once daylight savings time ends, maybe worth specifying it as 10:00 PT?

Reply:
Makes sense to me to change this to PT

Reply:
I'd personally prefer UTC on everything but I understand that's a lot easier for me (UK based) than the majority of maintainers :) I'd opt for uniformity in timezones used though, with PT being the _de facto_ standard, or at least have a primary one listed in PT and then maybe local one if maintainers are based elsewhere.

Reply:
@danielgblanco my understanding is that the problem is less around UTC, but around which daylight saving time adjustment we apply to our meetings, and there are no clear winners here, we choose PT so far, because most contributors are US-based. But depending on what we pick we always get different problems:

* If we pick UTC, everyone without DST adjustments in their timezone has their meeting moved twice per year
* If we pick a specific DST adjustment (currently PT), everyone within that timezone has no changes, everyone with a different schedule has 4 changes, and everyone without an adjusment in their timezone has moved their meeting twice per year. 

We can now choose to pick "PT" for the "US-friendly" meetings and a different time zone for "*-friendly" meetings

https://en.wikipedia.org/wiki/Daylight_saving_time_by_country has a nice picture to illustrate that mess.

Reply:
> We can now choose to pick "PT" for the "US-friendly" meetings and a different time zone for "*-friendly" meetings

this is what we did for Java: #2200

Reply:
I completely agree with the UTC+8. For example, I'm going to the LLM Sig tomorrow and I had to do a double-take as it is 7am not 8am at the moment. Ack there are some pragmatic impacts to this wrt attendance from west coast folks, just it is easier to get people to attend in the native time slot if they know what time it is without calculating backwards from california.

Reply:
https://github.com/open-telemetry/community/issues/2237